resources/views/mship/feedback/form.blade.php

```diff
@@ -5,106 +5,11 @@
 <script type="text/javascript">
   $(document).ready(function(){
     $('.datetimepickercustom').datetimepicker();
-
-    // Member search hook
-
-    var searchInput = $('#member-search-name');
-    var timeout;
-
-    // Bind input change for serch input
-    searchInput.bind("input", function(event){
-
-        $('#memberSearchSpinner').show();
-        $('#memberSearchNoResults').hide();
-        $('#memberSearchResults').hide();
-
-        clearTimeout(timeout);
-        timeout = window.setTimeout(function(){
-          $.get('{{route('mship.feedback.usersearch', "")}}/' + $('#member-search-name').val(), function(response){
```

Comment:
we should probably remove this route too?

Reply:
good spot! fixed